projects/kubernetes/kube-master.yml

```diff
@@ -51,18 +51,19 @@ services:
      - /dev:/dev
      - /etc/resolv.conf:/etc/resolv.conf
      - /lib/modules:/lib/modules
+     - /run:/run:rshared,rbind
```

Comment:
Re-reviewing #2131 it might be there is a symlink to `/var/lib` in there somewhere. I'll try this out (probably tomorrow at this stage).